src/Interface/InterfaceView/InterfaceView.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  KeyValue,
+  Col,
+  Row,
+  Checkbox
+} from '@folio/stripes/components';
+
+import css from './InterfaceView.css';
+
+
+export const InterfaceView = ({ item = {}, isNarrow = false }) => {
+  const columnCount = isNarrow ? 6 : 3;
```

Comment:
`columnCount` you use to define width of column, weird...

Reply:
I used different name